### PR TITLE
Using tag img on Image component

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
Use the tag `img` instead of `background-img` on Image component.

#### What problem is this solving?
We should use tag `img` for indexing on SEO.

#### How should this be manually tested?
[Access this workspace](https://theo--storecomponents.myvtex.com/)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Replace `bakground-image` for `img` on Image component.
```
Comment (luciannojunior) [inline CHANGELOG.md]: `background-image`**

File: react/productSummary.css [hunk 2]
```diff
@@ -5,7 +5,12 @@
 .imageContainer {}
 .information {}
 .element {}
-.image {}
+.imageInline {
+  max-height: 7rem;
+}
+.imageNormal {
+  max-height: 281px
```
Comment (luciannojunior) [inline react/productSummary.css]: Avoid `px`

File: react/productSummary.css [hunk 3]
```diff
@@ -5,7 +5,12 @@
 .imageContainer {}
 .information {}
 .element {}
-.image {}
+.imageInline {
+  max-height: 7rem;
```
Comment (luciannojunior) [inline react/productSummary.css]: Is this really necessary? Couldn't the boundaries be set in a parent component using Tachyons?

File: react/components/Image.js [hunk 4]
```diff
@@ -4,12 +4,11 @@ import PropTypes from 'prop-types'
...
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, description }) {
   return (
-    <div
-      className={`${className} ${productSummary.aspectRatio} w-100`}
-      style={{ backgroundImage: `url(${src})` }}
-    />
+    <div>
+      <img className={`${className}`} src={src} alt={description} />
```
Comment (Klynger) [inline react/components/Image.js]: No need for string interporlation here

File: react/productSummary.css (hunk 3 above)
Comment (theomoura) [reply react/productSummary.css]: I'm afraid it is. It need to be set directly on the image for IE compatibility.

File: react/components/Image.js [hunk 5]
```diff
@@ -4,12 +4,11 @@ import PropTypes from 'prop-types'
...
 
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, description }) {
   return (
-    <div
-      className={`${className} ${productSummary.aspectRatio} w-100`}
-      style={{ backgroundImage: `url(${src})` }}
-    />
+    <div>
```
Comment (Klynger) [inline react/components/Image.js]: I think this `div` is not necessary and removing this `div` the component would make any sense too.

File: react/components/Image.js [hunk 6]
```diff
@@ -4,12 +4,11 @@ import PropTypes from 'prop-types'
...
+      <img className={className} src={src} alt={description} />
```
Comment (klzns) [inline react/components/Image.js]: Why do we still need this component? It's the same as using `` directly

**Pull request merged**